<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/kde/icons/filesave.png</normaloff>:/kde/icons/filesave.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="undoButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/openicon/icons/edit-undo-8.png</normaloff>:/openicon/icons/edit-undo-8.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="redoButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="Experiment.qrc">
         <normaloff>:/openicon/icons/edit-redo-8.png</normaloff>:/openicon/icons/edit-redo-8.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>226</width>
        <height>315</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Scan</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBoxParameter"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="6" column="0">
          <widget class="QCheckBox" name="autoSaveCheckBox">
           <property name="toolTip">
            <string>Checkbox: automatically save file</string>
           </property>
           <property name="text">
            <string>Filename</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="MagnitudeSpinBox" name="startBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="stepsBox">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="scanTypeCombo">
           <item>
            <property name="text">
             <string>Linear Start to Stop</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Linear Stop to Start</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Randomized</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Direction</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="MagnitudeSpinBox" name="stopBox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Mode</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="scanModeComboBox">
           <item>
            <property name="text">
             <string>Single Scan</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Repeated Scan</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Step in place</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gate Set Scan</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="filenameEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QComboBox" name="comboBox_2">
           <property name="frame">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>steps</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>stepsize</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="rewriteDDSCheckBox">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Apply DDS settings from Main Ui after sequence is finished&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Rewrite DDS when finished</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>226</width>
        <height>315</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Evaluation</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Histogram bins</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="histogramBinsBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Eval Counter</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="counterSpinBox">
         <property name="maximum">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="integrateHistogramButton">
         <property name="text">
          <string>Integrate Histogram</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="errorBarCheckBox">
         <property name="text">
          <string>Error Bars</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Eval Method</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="evalMethodCombo"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QStackedWidget" name="evalStackedWidget">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>226</width>
        <height>315</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Timestamps</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QCheckBox" name="enableCheckBox">
         <property name="text">
          <string>Enable</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="saveRawDataCheckBox">
         <property name="text">
          <string>Save raw data</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Binwidth</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="MagnitudeSpinBox" name="binwidthSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>ROI start</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="MagnitudeSpinBox" name="roiStartSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>ROI width</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="MagnitudeSpinBox" name="roiWidthSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Counter</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="channelSpinBox">
         <property name="maximum">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QComboBox" name="integrateCombo">
         <item>
          <property name="text">
           <string>Integrate all</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Integrate each run</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>One trace per point</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>52</width>
           <height>141</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MagnitudeSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>MagnitudeSpinBox</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>undoButton</tabstop>
  <tabstop>redoButton</tabstop>
  <tabstop>comboBoxParameter</tabstop>
  <tabstop>startBox</tabstop>
  <tabstop>stopBox</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>stepsBox</tabstop>
  <tabstop>scanTypeCombo</tabstop>
  <tabstop>scanModeComboBox</tabstop>
  <tabstop>autoSaveCheckBox</tabstop>
  <tabstop>filenameEdit</tabstop>
  <tabstop>rewriteDDSCheckBox</tabstop>
  <tabstop>histogramBinsBox</tabstop>
  <tabstop>counterSpinBox</tabstop>
  <tabstop>integrateHistogramButton</tabstop>
  <tabstop>errorBarCheckBox</tabstop>
  <tabstop>evalMethodCombo</tabstop>
  <tabstop>enableCheckBox</tabstop>
  <tabstop>saveRawDataCheckBox</tabstop>
  <tabstop>binwidthSpinBox</tabstop>
  <tabstop>roiStartSpinBox</tabstop>
  <tabstop>roiWidthSpinBox</tabstop>
  <tabstop>channelSpinBox</tabstop>
  <tabstop>integrateCombo</tabstop>
 </tabstops>
 <resources>
  <include location="Experiment.qrc"/>
 </resources>
 <connections/>
</ui>
